### PR TITLE
probiotics.json

### PR DESCRIPTION
**Pull request by brianmackenna**
This is a proposed new measure. Description needs tweaking but can be tested on staging

### CODE REVIEW

**Comment by brianmackenna** [comment]
Please could this be reviewed before deploying to staging and final tweaks and checks?

File: openprescribing/frontend/management/commands/measure_definitions/probiotics [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "The Committee concluded that the evidence did not sufficiently demonstrate that the products are clinically effective. Additionally",
+    "the ACBS & MHRA consider the presentation of a product to prevent a clinical condition such as pouchitis as a specific medicinal claim.",
+    "Therefore these products now fall within the definition of a medicine (not a borderline substance)",
+    "and should be regulated as such. Importantly this means that they are also allowed on prescription, despite them not currently",
+    "being licnsed as a medicine and having been removed from the drug tariff as the evidence does not support them meeting the threshold",
```
Comment (sebbacon) [inline openprescribing/frontend/management/commands/measure_definitions/probiotics]: typo: licensed

File: openprescribing/measure_definitions/probiotics.json [hunk 2]
```diff
@@ -8,11 +8,12 @@
   ],
   "why_it_matters": [
     "The Advisory Committee on Borderline Substances (ACBS) is responsible for assessing and recommending products applying for listing in Part XV of the Drug Tariff. The ACBS recently reviewed some products listed in Part XV to ensure they are still clinically effective and appropriate to be listed here. They considered the probiotics VSL#3 and Vivomixx and advised the Secretary of State for Health and Social Care to remove these products. The Secretary of State accepted this recommendation and removed the products from the drug tariff in November 2018.",
-    "The Committee concluded that the evidence did not sufficiently demonstrate that the products are clinically effective. Additionally",
+    "This is consistent with <a href='https://www.england.nhs.uk/medicines/conditions-for-which-over-the-counter-items-should-not-routinely-be-prescribed/'> NHS England guidance </a> that recommends probitoics should nolonger be rotuinely prescribed.",
+    "<p> The Committee concluded that the evidence did not sufficiently demonstrate that the products are clinically effective. Additionally",
```
Comment (inglesp) [inline openprescribing/measure_definitions/probiotics.json]: You've added a `<p>` here, but there's no closing `</p>`.  You should probably also have `<p>`...`</p>` around the first paragraph too.

Comment (inglesp) [reply openprescribing/measure_definitions/probiotics.json]: I'll fix this now.

**Pull request merged**